config/initializers/secret_token.rb

```diff
@@ -1,11 +0,0 @@
-# Be sure to restart your server when you modify this file.
```

Comment:
how come you're removing this?  is it just a default in rails 5.2 now?

Reply:
it hasn't been used since Rails 4 (see http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html), the config/secrets.yml performs the similar function. So this file is unused and superfluous.

Reply:
This scheme has not been used since Rails 4.0, see http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html. so this file is unused and superfluous.